File: frontend/src/models/squadModel.ts
import type BaseModel from "./baseModel";
import type LeagueModel from "./leagueModel";

export default interface SquadModel extends BaseModel {
    name: string;
    userName: string;
    balanceCoin: number;
    image: string;
    leagueId: string;
    league: LeagueModel;
}

File: backend/Common/Domain/Models/SquadModel.cs
namespace Presentation.Common.Domain.Models;
public class SquadModel : BaseModel
{
    public string Name { get; set; }
    public string UserName { get; set; }

    public long BalanceCoin { get; set; }

    public string LeagueId { get; set; }
    public LeagueModel League { get; set; }
    
    public string Image { get; set; }

}
